docusaurus/docs/cloud/projects/notifications.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Notifications
+displayed_sidebar: cloudSidebar
+description: View your notifications on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/projects/notifications.html
+sidebar_position: 3
+---
+
+# Notifications
+
+The **Notification center** is accessible from the top navigation bar from anywhere in Cloud dashboard.
+
+![Projects page - List](/img/assets/cloud/notification-center.png)
+
+Currently, there are following notifications:
+
+* **Deploy completed**: When a deployment is successfully done.
+* **Build failed**: When deployment fails during the build stage.
+* **Deploy failed** When it fails during the deployment stage.
+* **Deploy triggered** When deployment is triggered by new push to the connected repo. This notificaiton isn't sent when you trigger a deployment manually.
+
+Clicking on a deploy-related notification card redirects you to **Log details** of the corresponding deployment.
```

Comment:
```suggestion
```
Moving this to the introduction instead.

Reply:
👍